.tekton/pull-request.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: spi-pull-request
+  annotations:
+    pipelinesascode.tekton.dev/on-event: "[pull_request]"
+    pipelinesascode.tekton.dev/on-target-branch: "[main]"
+    pipelinesascode.tekton.dev/max-keep-runs: "2"
+spec:
+  params:
+    - name: repo_url
+      value: "{{repo_url}}"
+    - name: revision
+      value: "{{revision}}"
+  pipelineRef:
+          name: docker-build
+          bundle: quay.io/redhat-appstudio/build-templates-bundle:v0.1.4    
+  pipelineSpec:
```

Comment:
only one can be used -> pipelineRef or pipelineSpec

Reply:
@Michkov made the necessary changes.